frontend/src/views/Seedlot/MySeedlots/SeedlotDataTable/index.tsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
+import { hashObject } from 'react-hash-string';
+
+import {
+  DataTable,
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableHeader,
+  TableBody,
+  TableCell,
+  Pagination,
+  Search
+} from '@carbon/react';
+
+import headerData from './data';
+import StatusItem from '../../../../components/StatusItem';
+import Participants from '../../../../components/Participants';
+import Seedlot from '../../../../types/Seedlot';
+
+interface HeaderType {
+  key: string,
+  header: string
+}
+
+interface DataTableInterface {
+  headers: HeaderType[],
+  onInputChange: any,
+  rows: any,
+  getHeaderProps: any,
+  getTableProps: any,
+}
+interface RowInterface {
+  cells: any,
+}
+
+interface SeedlotDataTableProps {
+  seedlots: Seedlot[]
+}
+
+const SeedlotDataTable = ({ seedlots }:SeedlotDataTableProps) => {
+  const [firstRowIndex, setFirstRowIndex] = useState(0);
+  const [currentPageSize, setCurrentPageSize] = useState(20);
+
+  return (
+    <>
+      <DataTable
+        rows={seedlots.slice(firstRowIndex, firstRowIndex + currentPageSize)}
+        headers={headerData}
+        isSortable
+      >
+        {({
+          rows, headers, onInputChange, getHeaderProps, getTableProps
+        }:DataTableInterface) => (
+          <TableContainer>
+            <Search
+              onChange={onInputChange}
+              placeholder="Search for seedlots"
+              size="lg"
+            />
+            <Table {...getTableProps()}>
+              <TableHead>
+                <TableRow>
+                  {headers.map((header:HeaderType) => (
+                    <TableHeader {...getHeaderProps({ header })}>
+                      {header.header}
+                    </TableHeader>
+                  ))}
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {rows.map((item:RowInterface) => (
+                  <TableRow key={hashObject(item.cells)}>
+                    <TableCell>{item.cells[0].value}</TableCell>
+                    <TableCell>{`${item.cells[1].value} class`}</TableCell>
+                    <TableCell>{item.cells[2].value}</TableCell>
+                    <TableCell>{item.cells[3].value}</TableCell>
+                    <TableCell>
+                      <StatusItem status={item.cells[4].value} />
+                    </TableCell>
+                    <TableCell>
+                      <Participants elements={item.cells[5].value} number={item.cells[0].value} />
```

Comment:
The number prop here is the number of participants, not the seedlot number.

Reply:
My mistake here, I understood wrongly the prop, you can leave as it is, I'll mark as resolved.